src/sentry/tsdb/base.py

```diff
@@ -276,7 +276,7 @@ def get_optimal_rollup_series(
             series.append(self.normalize_to_epoch(timestamp, rollup))
             timestamp = timestamp - timedelta(seconds=rollup)
```

Comment:
In this loop, `timestamp` is only being set backwards in time so it's guaranteed that we're appending to list in reverse order (so the list is descending)